### DISCUSSION TITLE
more helpful error message when movileprovisioning fails to verify

### DISCUSSION THREAD

**Discussion by kyubuns** [fastlane/fastlane#20167] [Ideas] [answered]
I ran into this error today in update_project_provisioning.

```
/Users/***/vendor/bundler/ruby/2.7.0/gems/fastlane-2.205.1/fastlane/lib/fastlane/actions/update_project_provisioning.rb:73:in `block (2 levels) in run': [!] undefined method `[]' for nil:NilClass (NoMethodError)
```

As it turns out, the cause was that mobile provisioning was set to Invalid.
However, I could not read anything from this error message and had to decipher the source code to see what values were wrong.

https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/update_project_provisioning.rb#L37
I'm not familiar with ruby, but apparently the return value of this p7.verify will tell me if it succeeded or not.
I thought that error messages could also be taken from somewhere.

(I wrote this in the discussion so that people with similar errors can find it by Google).

**Comment by lucgrabowski** [comment, marked as answer]
I've added a PR https://github.com/fastlane/fastlane/pull/20169

**Reply by kyubuns** [reply]
Thank you!
